packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -212,7 +212,7 @@ const typeGroups = {
   isosr: ['local', 'nfsiso', 'smb'],
 }
 
-const getSrPath = id => `/srs/${id}`
+const getSrPath = id => id !== undefined && `/srs/${id}`
```

Comment:
Does it work? The [condition](https://github.com/Rajaa-BARHTAOUI/xo-web/blob/new-sr-undefined-url/packages/xo-web/src/common/action-button.js#L99) checks for `undefined` so my guess is that it would still redirect but with "true" in the URL this time.